source/3ds/main.c

```diff
@@ -60,7 +60,11 @@ int main() {
 
     if (fileSelect("Load ROM", rom_name, "vb") < 0)
         goto exit;
+
+    #pragma GCC diagnostic push
+    #pragma GCC diagnostic ignored "-Wstringop-overflow="
     strncat(full_path, rom_name, 256);
```

Comment:
I guess I'm unsure if this is a bug or not. I think this warns because full_path is 256 bytes, and the limit defined is 256 bytes, and if you actually hit that limit you would actually truncate the null byte off the string. Ignored for now, 
but maybe this should just change the limit to 255